lua/luasnip/loaders/from_lua.lua

```diff
@@ -34,9 +34,9 @@ local function load_files(ft, files, add_opts)
 		local func_string = path_mod.read_file(file)
 
 		local load_ok, func = pcall(loadstring, func_string)
```

Comment:
But actually, now it would be best to just do `load_ok, func = loadfile(file)`, afaict this should work without any other changes.

Reply:
(Didn't do this before since we used to modify `func_string`, and I missed this when removing that)

Reply:
That works. Updating.

Reply:
I've updated the code to use loadfile in https://github.com/L3MON4D3/LuaSnip/commit/c6de1243b875cf8927c51843f13370d549fdb156

Reply:
Somehow it regressed. Load file load ok keeps being null. I'm going to revert to the old method.

Reply:
Right, ofc🤦
It would have to be `func, err_msg = loadfile...`, it does not match pcall exactly :/

Reply:
Right 🙈 , forgive me, it was late, and I assumed matching API's. It's working fine with loadfile: https://github.com/L3MON4D3/LuaSnip/pull/811/commits/c8bc64b9225b4a8253cb8595f5842388fb6b95b9

Reply:
same xD